account_invoice_kanban/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Account Invoice Kanban
+=======================
+
+
+Introduction
+============
+Currently Odoo just provides a status for Draft/open/paid status which is not
+enough to follow the full administrative process.
+
+This administrative process can be sometimes quite complex with workflow that
+might be different by customers.
+
+Some are usual  steps:
+
+* Timesheets created
+* Timesheets report internal validation
+* Timesheets Validated by the customer
+* ERP invoice created
+* ERP invoice validated
+* Other documents (contracts, official invoice) printing
+* Other documents (contracts, official invoice) delivery
+* PR/PO or contract reception for payment (some customers)
+* Payment received (=done)
+
+This list may vary over time and from customers.
+
+This module provides administrative stage management similar to the tasks in Odoo:
+
+* Add a Stage in Invoice (like in tasks) with the separate field for the additional
+  workflow on top of current one.
+* No button to switch from stages: simply clicking on it is enough (like tasks)
+* Add Kanban view, default with stage group by. 
+* Kanban becomes the default view.
+
+Configuration
+=============
+Stage are set up in the Invoicing application:
+
+#. Invoicing menu --> Configuration --> Stages --> Invoice Stages
+#. Create a new stage
+
+Usage
+=====
+#. Go to Invoicing menu --> Customers --> Customer Invoices
+#. Enjoy the new kanban view 
+#. Drag and drop the invoices from stage to stage
+#. In the form view, click on any new stage to change it
+
+Known Issues / Roadmap
+======================
+* Stages are somehow redondant with invoice status: it would be good to have the option to
```

Comment:
[We solved this with a base module too](https://github.com/OCA/server-tools/tree/10.0/base_kanban_stage_state)